### PR TITLE
fix security issues related to routing

### PR DESCRIPTION
**Pull request by Battula**
[ME-184110410](https://www.pivotaltracker.com/story/show/184110410)

### CODE REVIEW

**Review by jacobkagon — APPROVED** [review]
Disables non html formats. Change looks good!

File: config/routes.rb [hunk 1]
```diff
@@ -300,8 +300,7 @@
     end
 
     resources :inboxes, only: [:new, :create, :show, :destroy]
-    resources :families, only: [:show] do
```
Comment (TonyHasIdeas) [inline config/routes.rb:303]: Do we need the :show action at all?

**Comment by markgoho** [comment]
@Battula the ticket you've referenced says "Temporary File Download" and talks about temporary files being left in the production environment

I'm not sure how this change relates to that ticket

**Review by TonyHasIdeas — APPROVED** [review]
LGTM

File: config/routes.rb (hunk 1 above)
Comment (Battula) [reply config/routes.rb:303]: No, that isn't being used at all

**Review by jayreddy519 — APPROVED** [review]

**Pull request merged**